### PR TITLE
Add data tables to api view

### PR DESCRIPTION
**Pull request by chidozieononiwu**
UI Upgrades to APIView

### CODE REVIEW

File: src/dotnet/APIView/APIViewWeb/Repositories/ReviewManager.cs [hunk 1]
```diff
@@ -492,7 +492,7 @@ public async void UpdateReviewBackground()
             // Enabling this only for manual reviews in the beginning to check impact on system performance
             // We will enable it for all reviews based on the perf details
             // Automatic reviews are already updated as part of scheduled upload daily
-            var reviews = await _reviewsRepository.GetReviewsAsync(false, "All");
+            var reviews = await _reviewsRepository.GetReviewsAsync(false, "Algggl");
```
Comment (benbp) [inline src/dotnet/APIView/APIViewWeb/Repositories/ReviewManager.cs:495]: Intentional?

Comment (chidozieononiwu) [reply src/dotnet/APIView/APIViewWeb/Repositories/ReviewManager.cs:495]: Yes

**Pull request closed without merging**